src/main/java/uk/gov/hmcts/reform/pip/data/management/service/PublicationService.java

```diff
@@ -106,4 +109,31 @@ public String getByArtefactId(UUID artefactId, Boolean verification) {
             throw new NotFoundException(String.format("No artefact found with the ID: %s", artefactId));
         }
     }
+
+    /**
+     * Enforces conditional mandatory fields based on the publication type. This is to ensure that status updates are
+     * able to persist indefinitely if required.
+     */
+    public boolean validateDateFromDateTo(LocalDateTime displayFrom, LocalDateTime displayTo, ArtefactType type) {
+
+        ArrayList<ArtefactType> mandatoryDateArtefactTypes = new ArrayList<>();
+        mandatoryDateArtefactTypes.add(ArtefactType.LIST);
+        mandatoryDateArtefactTypes.add(ArtefactType.OUTCOME);
+        mandatoryDateArtefactTypes.add(ArtefactType.JUDGEMENT);
```

Comment:
this seems like a static rule that doesnt need to be created each time the method is called

Reply:
could even just use the ArtefactType enum

Reply:
fixed by validation service